locations/spiders/tobu_park_jp.py

```
@@ -0,0 +1,52 @@
+# found rather than picking a single amenity for the whole page.
+CATEGORY_MAP = {
+    "car-parking": Categories.PARKING,
+    "bicycle-parking": Categories.BICYCLE_PARKING,
+    "bike-parking": Categories.MOTORCYCLE_PARKING,
+    "bicycle-rental": Categories.BICYCLE_RENTAL,
+}
+
+
+class TobuParkJPSpider(SitemapSpider):
+    name = "tobu_park_jp"
+    item_attributes = {"brand": "TOBU PARK", "brand_wikidata": "Q131275047"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)